test/expect/TestFuser.test_lstm_cuda-backward.expect

```diff
@@ -1,111 +0,0 @@
-graph(%0 : Float(*, *),
```

Comment:
let's keep this expect file and the forward expect file for now by not asserting failures?